spec/features/checkout_spec.rb

```diff
@@ -31,7 +31,7 @@
 
       # select default shipping
       click_button "Save and Continue"
-      expect(page).to have_content("Shipping total: $68.81")
+      expect(page).to have_content("$68.81")
```

Comment:
Any idea why this is failing?

Reply:
Because of the text, it’s displaying `UPS shipping method` instead of `Shipping total`, and that’s weird because only on one version fails

Reply:
@forkata this is the error: https://travis-ci.org/solidusio-contrib/solidus_active_shipping/jobs/357565410

Reply:
I retried the job that was failing and it succeeded, so this must be a flaky test https://travis-ci.org/solidusio-contrib/solidus_active_shipping/builds/357565408

Reply:
done, I removed this change